sympy/physics/units.py

```diff
@@ -248,6 +248,8 @@ def free_symbols(self):
 G = gravitational_constant = Rational('6.67428') * ten**-11 * m**3 / kg / s**2
 u0 = magnetic_constant = 4*pi * ten**-7 * N/A**2
 e0 = electric_constant = 1/(u0 * c**2)
+Ke = coulombs_constant = 1/(4*pi * e0)
+R = gas_constant = Rational('8.3144621') * J / mol / K
```

Comment:
Isn't the gas constant just defined as Avogadros number times Boltmannz's constant?